src/trendlines/static/core.js

```diff
@@ -25,8 +25,8 @@ function populateTree(data) {
     // jsTree puts the original data structure in a nested object
     // called 'original'. How original of them. Hahaha I crack myself up.
     // If `url` is defined, take us there.
-    if (data.node.original.url !== null) {
-      var expected = data.node.original.url;
+    if (data.node.original.metric_id !== null) {
+      var expected = "/plot/" + data.node.original.id;
```

Comment:
Do we want to use `text` instead of `id`? Right now, both are strings and both are the name of the metric, but in the future I think I would like:
+ `id` to be the metric's internal id
+ `text` only be the last item in the name. eg: 
    ```
    + foo         # foo
      + bar       # foo.bar
    + bar
      + baz
        + biz     # bar.baz.biz
```

Reply:
I'll leave it for now.